app/controllers/smart_proxies_controller.rb

```diff
@@ -64,6 +64,7 @@ def puppet_dashboard
     dashboard = Dashboard::Data.new("puppetmaster = \"#{@smart_proxy.name}\"")
     @hosts = dashboard.hosts
     @report = dashboard.report
+    @latest_events = dashboard.latest_events
```

Comment:
If this is done, I'd rather move it to a form object of sorts, so we just pass one instance variable to the view instead of 4. Thus not breaking Sandi Metz's rule 4 https://robots.thoughtbot.com/sandi-metz-rules-for-developers